docs/workflow.rst

```diff
@@ -238,12 +233,13 @@ Let's get in more detais:
 
```

Comment:
The successful start is when you have both these remotes on your local development: => on your local repository.